docs/src/Makefile

```diff
@@ -32,15 +32,18 @@ help:
 
 clean:
 	-rm -rf $(BUILDDIR)/*
+	-rm -rf generated/*
+	-rm -rf modules/generated/
+	-rm -rf auto_examples/*
 
 html:
 	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
 	rm -r $(BUILDDIR)/html/_sources
-	cp -r $(BUILDDIR)/html/* ../
+	cp -r $(BUILDDIR)/html/* .
```

Comment:
Why this change?

Reply:
My mistake, I don't like the copying to the upper level though. Why are we doing it in the first place?